### PR TITLE
Update ZIKA.md

### PR DESCRIPTION
**Pull request by victorlin**
### Description of proposed changes

Changes identified when running through the 2022-03-25 update with @j23414.

### Related issue(s)

_N/A_

### Testing

_N/A_

### CODE REVIEW

**Review by j23414 — APPROVED** [review]
Looks correct, especially noting to set env variables, and transforming some of the intermediate files. Thanks!

File: builds/ZIKA.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # ZIKA Pipeline Notes
 
+## Setup
+
+1. Start fauna rethinkdb server in a separate terminal.
+2. Make sure environment variables for connecting to fauna are set.
```
Comment (tsibley) [inline builds/ZIKA.md:6]: Maybe worth pointing out in the README here that 1 is not relevant to our own builds since there's a central fauna server?

Comment (j23414) [reply builds/ZIKA.md:6]: Huh, I just tried it and the pipeline actually does work without running `rethinkdb` in a separate terminal. Thanks! I was doing using the combination of `rethinkdb` and `chateau` to view the changed database. Yes, I vote for dropping step 1.

Comment (victorlin) [reply builds/ZIKA.md:6]: #117

**Pull request merged**